docs/vector_store.ipynb

```diff
@@ -249,6 +252,7 @@
     "1. `instance_id` - The name of the Spanner instance\n",
     "1. `database_id` - The name of the Spanner database\n",
     "1. `table_name` - The name of the table within the database to store the documents & their embeddings.\n",
+    "1. `row_id` - The name of the row identifier column of the table\n",
```

Comment:
Instead of making the next 2 changes, we should just comment out the customization options in the init table call:
```python
SpannerVectorStore.init_vector_store_table(
    instance_id=INSTANCE,
    database_id=DATABASE,
    table_name=TABLE_NAME,
    # Customize the table creation
    # id_column="row_id",
    # metadata_columns=[
    #     TableColumn(name="metadata", type="JSON", is_null=True),
    #     TableColumn(name="title", type="STRING(MAX)", is_null=False),
    # ],
    # secondary_indexes=[
    #     SecondaryIndex(index_name="row_id_and_title", columns=["row_id", "title"])
    # ],
)
```

Reply:
metadata_columns is expected in the current function, so commenting this time will throw an error. I will file a bug for this..